table/tables/tables.go

```diff
@@ -323,6 +323,20 @@ func (t *Table) rebuildIndices(ctx context.Context, h int64, touched []bool, old
 	return nil
 }
 
+func (t *Table) delDirtyIndexKey(createdIdx int, recordID int64, r []interface{}, txn kv.Transaction) {
+	for i, v := range t.indices {
+		if v == nil {
+			continue
+		}
+
+		if i >= createdIdx {
+			break
+		}
+		colVals, _ := v.FetchValues(r)
+		_ = v.X.Delete(txn, colVals, recordID)
```

Comment:
I have some troubles that I check the index are all available to create, then I create the indices, but the check and create is not a atomic operation, so I may fail when I create one of the indices which is created by other people in the interval..